src/Tile/Tile.Component.js

```diff
@@ -74,24 +74,50 @@ export const TileComponent = () => {
                 </Tile>
             </Example>
 
+            <Example
+                title='Media with Actions Tile'>
+                <Tile>
+                    <Tile.Media>
+                        <Image photo='https://placeimg.com/400/400/nature' size='l'
+                            type='circle' />
+                    </Tile.Media>
+                    <Tile.Content title='Tile Title' />
+                    <Tile.Actions>
+                        <Popover
+                            body={
+                                <Menu>
+                                    <Menu.List>
+                                        <Menu.Item url='/'>Option 1</Menu.Item>
+                                        <Menu.Item url='/'>Option 2</Menu.Item>
+                                        <Menu.Item url='/'>Option 3</Menu.Item>
+                                        <Menu.Item url='/'>Option 4</Menu.Item>
+                                    </Menu.List>
+                                </Menu>
+                            }
+                            control={<Button glyph='vertical-grip' option='light' />}
+                            placement='bottom-end' />
+                    </Tile.Actions>
+                </Tile>
+            </Example>
+
             <Example
                 title='Product Tile'>
                 <div>
-                    <ProductTile role='button'>
-                        <ProductTile.Media image='https://techne.yaas.io/images/product-thumbnail-wide.png' />
-                        <ProductTile.Content title='Tile Title'>
+                    <Tile productTile role='button'>
```

Comment:
When we're allowing the productTile to be used as a button, should we swap out for actual html button syntax? This is more a question for fundamental-styles but we should probably think about it.

Reply:
I think this entire concept of tiles being buttons needs to be addressed in fd-styles and fd-react going forward.  The current props/implementation seems half-baked.

Reply:
Sounds good - looks like 4:1 ratio is good for header image and will switch it to nature 👍

Reply:
@jbadan @greg-a-smith should I comment this as a topic for our thursday meeting?